site/layouts/_default/docs.html

```diff
@@ -16,9 +16,14 @@ <h5 class="offcanvas-title" id="bdSidebarOffcanvasLabel">Browse docs</h5>
     <main class="bd-main order-1">
       <div class="bd-intro pt-2 ps-lg-2">
         <div class="d-md-flex flex-md-row-reverse align-items-center justify-content-between">
-          <a class="btn btn-sm btn-bd-light mb-3 mb-md-0 rounded-2" href="{{ .Site.Params.repo }}/blob/v{{ .Site.Params.current_version }}/site/content/{{ .Page.File.Path | replaceRE `\\` "/" }}" title="View and edit this file on GitHub" target="_blank" rel="noopener">
-            View on GitHub
-          </a>
+          <div class="mb-3 mb-md-0 d-flex">
```

Comment:
Oh yes nice catch. You can suggest a `.text-no-wrap` in Bootstrap to avoid:

![Screenshot 2023-01-04 at 13 43 14](https://user-images.githubusercontent.com/17381666/210557579-b3bb61e1-69dc-4146-93b2-ba0db1072f61.png)

Reply:
See https://github.com/twbs/bootstrap/pull/37803